bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutator.java

```diff
@@ -61,6 +73,26 @@
     }
   }
 
+  private static class MutationOperation {
+    final Mutation mutation;
+    final long operationId;
+    final boolean isCloseMarker;
+
+    public MutationOperation(boolean isCloseMarker) {
```

Comment:
nit:
static MutationOperation CloseMarker() {
  MutationOperation(x, y, true);
}

public MutationOperation(Mutation mutation, long operationId) {
  MutationOperation(mutation, operationId, false);
}

private MutationOperation(Mutation mutation, long operationId, boolean isCloseMarker) {
       this.mutation = mutation;
       this.operationId = operationId;
       this.isCloseMarker = isCloseMarker;
     }